templates/DBFile_image.ss

```diff
@@ -1 +1 @@
-<img src="$URL.ATT" alt="$Title.ATT" />
+<img src="$URL.ATT" alt="$Title.ATT" $AttributesHTML />
```

Comment:
Maybe we want all the attributes to be rendered with `$AttributesHTML`. Not sure yet.

Reply:
I'd say all attributes in this case.  Should either do all within PHP or all within the template, not both IMO